templates/base.html

```diff
@@ -4,9 +4,12 @@
 <meta name=description content="{% block meta_description %}{% endblock %}">
 
 <link rel=stylesheet href=https://rawgit.com/cafebabel/styleguide/master/styleguide.css>
+<link rel=stylesheet href=/static/css/styles.css>
 <link rel=stylesheet href=/static/css/edit.css>
 <script src=/static/js/utils.js></script>
 
+<body class="{% block body_class%}{% endblock %}">
```

Comment:
for consistency, insert a space `…_class %}`

Reply:
ok